catalog/dags/data_refresh/copy_data.py

```diff
@@ -198,7 +198,7 @@ def copy_data(
 def copy_upstream_table(
     upstream_conn_id: str,
     downstream_conn_id: str,
-    environment: Environment,
+    target_environment: Environment,
```

Comment:
`target_environment` is not used anywhere in this function, should it be removed?